### PR TITLE
Checkout before confirm order

### PR DESCRIPTION
**Pull request by mvn-vyle-dn**
## Description
- Show detail order before submit order
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
LGTM

File: public/js/public/cart.js [hunk 1]
```diff
@@ -125,8 +129,14 @@ function modifyCart(id, option) {
...
                     case 'change':
-                        var number = $('#number-product-' + id).val();
-                        if(number <=0) modifyCart(id, 'delete');
+                        var number, numberCheckout, numberCart;
+                        if(window.location.pathname == '/checkout/cart') {
+                            numberCheckout = parseInt($('#cart-detail-checkout .box-cart-detail .box-cart-scroll table tbody tr td span .number-product-' +id).val());
+                        }
+                        numberCart = parseInt($('#number-product-' +id).val());
+                        number = numberCart != cart[j].quanlity ? numberCart : numberCheckout;
+                        console.log('a '+numberCheckout+ ' ' +numberCart);
```
Comment (at-quangtran) [inline public/js/public/cart.js]: xoá console

File: public/js/public/checkout.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    $.ajax({
+        url: "/api/users/info",
+        type: "get",
+        headers: {
+            'Accept': 'application/json',
+            'Authorization': 'Bearer ' + token
+        },
+        success: function(response) {
+            var user = response.result;
+          $('#NGUOINHAN').val(user.full_name);
```
Comment (at-quangtran) [inline public/js/public/checkout.js]: tiếng anh hết nhé

File: resources/views/public/pages/cart.blade.php [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+        </div>
+      </div>
+      <div class="hinh-thuc col-sm-6">
+        <h3 class="title text-capitalize">
+          <i class="fa fa-money"></i> {{__('user/cart.payments')}}
+        </h3>
+        <span class="field-validation-valid" data-valmsg-for="NGAYGIAO" data-valmsg-replace="true"></span>
+        <div class="form-group">
+          <div class="input-group date" id="NgayGiao">
+            <input value="" class="form-control" data-val="true" data-val-date="The field NGAYGIAO must be a date." data-val-required="Vui lòng nhập chính xác NGÀY VÀ GIỜ giao hàng" id="NGAYGIAO" name="NGAYGIAO" placeholder="{{__('user/cart.delivery_date')}}" type="text">
```
Comment (at-quangtran) [inline resources/views/public/pages/cart.blade.php]: mấy cái message ni viết thẳng luôn hả?

File: resources/views/public/pages/cart.blade.php [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+      </div>
+    </form>
+  </div>
+</div>
+<div class="modal fade" id="modalbank" tabindex="-1" role="dialog" aria-labelledby="modalbank" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></button>
+                <h3 class="modal-title text-center text-uppercase lead" id="modalbank">Thông tin chuyển khoản</h3>
```
Comment (at-quangtran) [inline resources/views/public/pages/cart.blade.php]: message

File: public/js/public/cart.js [hunk 5]
```diff
@@ -80,26 +80,30 @@ function showCart(cart) {
...
+        changeNumberCart(cart.length);
         $('.box-cart').html(html);
     }else {
         collapseCart();
         $('.box-cart').html('<p class="title text-uppercase">Your Cart is empty</p>');
         changeNumberCart(cart);
     }
+    $('#cart-detail-checkout .box-cart-detail p').remove();
+    $('#thanhtien').html(total.toFixed(3));
+    $('#phivc').html(formatNumber(50000));
```
Comment (at-quangtran) [inline public/js/public/cart.js]: tiếng anh, const 50000

**Review by mvn-quangtran-dn — APPROVED** [review]
LGTM

**Pull request merged**